modules/widgets/internet-defense-league.php

```diff
@@ -0,0 +1,154 @@
+<?php
+
+class Jetpack_Internet_Defense_League_Widget extends WP_Widget {
+
+	public $defaults = array();
+
+	public $variant;
+	public $variants = array();
+
+	public $campaign;
+	public $campaigns  = array();
+	public $no_current = true;
+
+	public $badge;
+	public $badges = array();
+
+	function __construct() {
+		parent::__construct(
+			'internet_defense_league_widget',
+			/** This filter is documented in modules/widgets/facebook-likebox.php */
+			apply_filters( 'jetpack_widget_name', esc_html__( 'Internet Defense League', 'jetpack' ) ),
+			array(
+				'description' => __( 'Show your support for the Internet Defense League.', 'jetpack' ),
+				'customize_selective_refresh' => true,
+			)
+		);
+
+		// When enabling campaigns other than 'none' or empty, change $no_current to false above.
```

Comment:
All the code associated to `$no_current` being `false` is never used since there's no way for user to change the property, which is always `true`, so maybe it should be removed.